b2sdk/sync/policy.py

```diff
@@ -9,18 +9,32 @@
 ######################################################################
 
 from abc import ABCMeta, abstractmethod
+from enum import Enum
 
 import six
 import logging
 
-from ..exception import CommandError, DestFileNewer
+from ..exception import DestFileNewer
 from .action import LocalDeleteAction, B2DeleteAction, B2DownloadAction, B2HideAction, B2UploadAction
+from .exception import InvalidArgument
 
 ONE_DAY_IN_MS = 24 * 60 * 60 * 1000
 
 logger = logging.getLogger(__name__)
 
 
+class NewerFileSyncMode(Enum):
+    SKIP = 101
```

Comment:
https://docs.python.org/3.4/library/enum.html#autonumber ?

Reply:
Brian suggested we use strings instead of integers. Should we go with strings or we can use autonumber but autonumber might produce same values for difference Enums.
`NewerFileSyncMode.SKIP.value == CompareVersionMode.MODTIME.value` might return `True`.

Reply:
Brian suggested it before we discussed Enums.

Comparison that you have mentioned is not a threat, as enums should not be compared by with using `.value` (or it will cause exactly what you mentioned)